src/main/java/org/folio/circulation/domain/Loan.java

```diff
@@ -549,4 +555,16 @@ public Loan markItemMissing(String comment) {
 
     return closeLoan(MISSING, comment);
   }
+
+  public FeeAmount getRemainingFeeFineAmount() {
```

Comment:
Is this the sum of all of the outstanding balances for fees that relate to this loan?

Reply:
Yes, correct.